components/script/dom/document.rs

```diff
@@ -3102,14 +3100,10 @@ impl DocumentMethods for Document {
             return Err(Error::Security);
         }
 
-        let header = Header::parse_header(&[cookie.into()]);
-        if let Ok(SetCookie(cookies)) = header {
-            let cookies = cookies.into_iter().map(Serde).collect();
-            let _ = self.window
-                        .upcast::<GlobalScope>()
-                        .resource_threads()
-                        .send(SetCookiesForUrl(self.url(), cookies, NonHTTP));
-        }
+        let _ = self.window
+                    .upcast::<GlobalScope>()
+                    .resource_threads()
+                    .send(SetCookiesForUrl(self.url(), String::from(cookie), NonHTTP));
```

Comment:
Oh I see now why things have been renamed. What about renaming `SetCookieForUrl` to `SetCookieHeaderForUrl` and keep `SetCookiesForUrl` as is?